src/auth/screens/language-settings.js

```diff
@@ -35,6 +35,10 @@ export default [
     name: 'Русский',
   },
   {
+    code: 'eo',
+    emojiCode: ':star:',
```

Comment:
I think this should probably be a flag, even if it's `:waving_white_flag:`.

Reply:
Yeah @lex111 and i have discussed how we can possibly change this, but in the meantime `:waving_white_flag:` should do :)